docs/docs/reference/command-line-interface.md

```diff
@@ -86,6 +86,38 @@ meltano add --custom extractor tap-covid-19
 docker run --interactive -v $(pwd):/project -w /project meltano/meltano add --custom extractor tap-covid-19
 ```
 
+To add a plugin from a [plugin definition](/concepts/project#custom-plugin-definitions) YAML file as a [custom plugin](/concepts/plugins#custom-plugins), use the `--from-ref` option referencing a URL or local path:
```

Comment:
My preference would be to have this below the --inherit-from explanation as it's a more rare use case. 

I'd also like a bit of context for why users would likely want to use this. Could we add a short statement about when this functionality is useful. And perhaps we could have a separate issue and PR to add a mention in https://docs.meltano.com/tutorials/custom-extractor/ ? 

I'll approve so it's non-blocking as this is a relatively minor docs update.

Reply:
https://github.com/meltano/meltano/pull/7907/commits/263cc848d90de07744b950c1f07d967a119d3667
https://github.com/meltano/meltano/pull/7907/commits/57d4fb05b81601d8ea4a0b13564e9e07b4a95227